src/vm/sampleprofiler.cpp

```diff
@@ -28,9 +28,12 @@ BYTE* SampleProfiler::s_pPayloadManaged = NULL;
 CLREventStatic SampleProfiler::s_threadShutdownEvent;
 unsigned long SampleProfiler::s_samplingRateInNs = 1 MILLION; // 1ms
 bool SampleProfiler::s_timePeriodIsSet = FALSE;
+
+#ifndef FEATURE_PAL
 PVOID SampleProfiler::s_timeBeginPeriodFn = NULL;
 PVOID SampleProfiler::s_timeEndPeriodFn = NULL;
 HINSTANCE SampleProfiler::s_hMultimediaLib = NULL;
+#endif //FEATURE_PAL
 
 typedef MMRESULT (WINAPI *TimePeriodFnPtr) (UINT uPeriod);
```

Comment:
TimePeriodFnPtr needs to be behind this as well, since it won't exist on Unix.

Reply:
Oh, oops. Thanks!